CMakeLists.txt

```diff
@@ -485,6 +556,7 @@ add_definitions(-DCRYPTOPP_DISABLE_SHANI)
 # Fix failing builds after commit
 # https://github.com/weidai11/cryptopp/commit/df9fa62205f2d341e2b1b26595a3a1b6377c60c5
 add_definitions(-DCRYPTOPP_DISABLE_CLMUL)
+add_definitions(-DCRYPTOPP_DISABLE_ASM)
```

Comment:
If we are going disable functionality that otherwise looks good to have then we need explain why we are disabling it.

Reply:
This was a workaround for cross-compiling, I'll remove it.